template/README.mustache

```diff
@@ -182,14 +182,16 @@ public class FingerprintApiExample {
         // Search events with custom filters
         try {
             // By visitorId
-            EventsGetResponse response = api.searchEvents(LIMIT, FPJS_VISITOR_ID, null, null, null, null, null, null, null);
+            SearchEventsResponse response = api.searchEvents(LIMIT, new FingerprintApi.SearchEventsFilter().setVisitorId(FPJS_VISITOR_ID));
+            // Next page
+            // SearchEventsResponse response = api.searchEvents(LIMIT, new FingerprintApi.SearchEventsFilter().setPaginationKey(response.getPaginationKey()).setVisitorId(FPJS_VISITOR_ID));
             // Bad bot
-            // EventsGetResponse response = api.searchEvents(LIMIT, null, "bad", null, null, null, null, null, null);
+            // SearchEventsResponse response = api.searchEvents(LIMIT, new FingerprintApi.SearchEventsFilter().setBot("bad"));
             // Filtered by IP
-            // EventsGetResponse response = api.searchEvents(LIMIT, null, null, "192.168.0.1/32", null, null, null, null, null);
+            // SearchEventsResponse response = api.searchEvents(LIMIT, new FingerprintApi.SearchEventsFilter().setIpAddress("192.168.0.1/32"));
             System.out.println(response.getProducts().toString());
```

Comment:
This line seems out of date. `EventsGetResponse` has `getProducts` but `SearchEventsResponse` does not as far as I can tell.

Reply:
Fixed